modular_robot_simulation/revolve2/modular_robot_simulation/_build_multi_body_systems/_builders/__init__.py

```diff
@@ -17,5 +19,7 @@
     "Builder",
     "CameraSensorBuilder",
     "CoreBuilder",
+    "DroneCoreBuilder",
```

Comment:
this should just be a core builder -> no need for a drone core builder as it still is a box